src/server/components/proxyMiddleware/router.ts

```diff
@@ -16,20 +17,22 @@ const middleware = [...bodyParser, singleRouteCsrf, addCsrfTokenToLocals];
 
 export const applyRouter = express.Router();
 
-applyRouter.get("/application/lawyers/start", (req: Request, res: Response) => {
-  req.session.application ??= {};
-  res.render("apply/lawyers/start");
-});
-applyRouter.get("/application/lawyers/which-list-of-lawyers", middleware, (req: Request, res: Response) => {
-  res.render("apply/lawyers/which-list-of-lawyers", { countriesList, answer: req.session.application?.country });
-});
-applyRouter.post("/application/lawyers/which-list-of-lawyers", middleware, lawyersPostController);
-applyRouter.get("/application/lawyers/not-currently-accepting", (req: Request, res: Response) => {
-  res.render("apply/not-accepting-currently", {
-    backLink: "/application/lawyers/which-list-of-lawyers",
-    country: req.session?.application?.country,
-  });
-});
+applyRouter.get("/application/:serviceType(lawyers|funeral-directors)/start", Controllers.getStartPageController);
+applyRouter.get(
+  "/application/:serviceType(lawyers|funeral-directors)/which-country-list-do-you-want-to-be-added-to",
+  middleware,
+  Controllers.getCountrySelectPageController
```

Comment:
This is fine to leave as it is since some of the codebase already uses this pattern. 

This is just an FYI. Let me know if you want a more detailed explanation though. Some of the code has conflated route handlers with controllers. Better naming of this would be `Controllers.countrySelect.get`, or `Controllers.countrySelect.post`. Alternatively `CountrySelectController.get` `CountrySelectController.post`. 

There is some context duplication when using something like `Controllers.countrySelectController` - I already know I'm in Controllers, so no need to tell me again. https://github.com/kettanaito/naming-cheatsheet#avoid-context-duplication

Reply:
Yeah I think I might leave it the way it is since it is a pattern that is already used in this project, but this way interests me `Controllers.countrySelect.get`. How would that work from a code point of view? A function called `get()` nested in an object called `countrySelect`?

Reply:
Ah I wasn't aware you'd already done this in the project.

Okay in that case I'd love to know your thoughts on my changes. I'm not a fan of creating a new file for just one route handler so I've put everything in an object.

Let me know what you think of this approach.

Reply:
For all of these cases it would be one file, two route handlers. (get handler and post handler). It then also means that the directory and files roughly translate to the api routes. 

I can understand why one method or handler per file may be annoying to navigate but the alternative has maintainability issues. Let me know what else you're not a fan of, though. I can have a think on other solutions.